### PR TITLE
Keras model changes

### PR DESCRIPTION
**Pull request by anj-s**
Added L2 reg loss, training argument for Keras BatchNorm layers.
Explicitly added BatchNorm updates during training.

### CODE REVIEW

**Comment by anj-s** [comment]
Realized that I did not merge this into the resnet-perf-tweaks branch. Once this is merged I am going to merge the keras/ folder changes into the master branch of model garden.
